### PR TITLE
Add lmdbxx package

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: cmake/projects/lmdbxx/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME lmdbxx
+  VERSION "0.9.14.0"
+  URL "https://github.com/isaachier/lmdbxx/archive/hunter-0.9.14.0-p1.tar.gz"
```
Comment (isaachier) [inline cmake/projects/lmdbxx/hunter.cmake]: Let me fix this.

File: cmake/projects/lmdb/hunter.cmake [hunk 2]
```diff
@@ -19,8 +19,17 @@ hunter_add_version(
...
-    )
-  
+  )
+
+hunter_add_version(
+    PACKAGE_NAME
+    lmdb
+    VERSION
+    0.9.21-p2
+    URL "https://github.com/hunter-packages/lmdb/archive/v0.9.21-p2.tar.gz"
```
Comment (isaachier) [inline cmake/projects/lmdb/hunter.cmake]: Wait nvm already fixed here.

**Comment by isaachier** [comment]
Wait I said I already fixed it. That was the first commit.

File: cmake/configs/default.cmake [hunk 3]
```diff
@@ -228,7 +228,8 @@ hunter_config(libsodium VERSION 1.0.10)
 hunter_config(libuv VERSION 1.14.0-p1)
 hunter_config(libxml2 VERSION 2.9.4)
 hunter_config(libyuv VERSION 1514-p3)
-hunter_config(lmdb VERSION 0.9.21-p1)
+hunter_config(lmdb VERSION 0.9.21-p2)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: Please make separate pull request with this update.
